cms/templates/index.html

```diff
@@ -35,9 +36,8 @@ <h3 class="sr">${_("Page Actions")}</h3>
           % endif
 
           % if is_programs_enabled:
-            <!-- TODO: Link to the program creation view in the authoring app. -->
-            <button class="button new-button new-program-button"><i class="icon fa fa-plus icon-inline"></i>
-            ${_("New Program")}</button>
+            <a href=${reverse('programs') + 'new'} class="button new-button new-program-button"><i class="icon fa fa-plus icon-inline"></i>
```

Comment:
Yes, that's cleaner and consistent with what I'm doing in the `ProgramAuthoringView`. Updated.